common.py

```diff
@@ -6,8 +6,13 @@
 from types import TracebackType
 import http.server
 from multiprocessing import Process
+from git.repo import Repo
+import shutil
+import shlex
 import host
+from k8sClient import K8sClient
 from logger import logger
+import reglocal
```

Comment:
I think there should be an order/hierarchy in which modules depend on each other.

`logger.py` is lowest, then `host.py`, then `common.py`, then everything else.

That means, `common.py` cannot include modules except `logger.py` and `host.py` (and third party pip modules and stdlib).

Currently,
- `reglocal.py` only uses `host.py` and `k8sClient.py`
- `k8sClient.py` uses `common.py`.
that is all good, but if now `common.py` imports `reglocal.py`, there is a cycle.

Some of the code (that requires `reglocal` or `k8sClient`) should move to another module.